transactions/transactor_test.go

```diff
@@ -135,80 +128,29 @@ func (s *TxQueueTestSuite) TestCompleteTransaction() {
 		AccountKey: &keystore.Key{PrivateKey: key},
 	}
 	testCases := []struct {
-		name       string
-		gas        *hexutil.Uint64
-		gasPrice   *hexutil.Big
-		signTxArgs *sign.TxArgs
+		name     string
+		gas      *hexutil.Uint64
+		gasPrice *hexutil.Big
 	}{
 		{
 			"noGasDef",
 			nil,
 			nil,
-			s.defaultSignTxArgs(),
 		},
 		{
 			"gasDefined",
 			&testGas,
 			nil,
-			s.defaultSignTxArgs(),
 		},
 		{
 			"gasPriceDefined",
 			nil,
 			testGasPrice,
-			s.defaultSignTxArgs(),
-		},
-		{
```

Comment:
I think most of these test cases still make sense. They check various params passed in the params and they should be checked.

Reply:
These tests were removed because we removed `Transactor.mergeSignTxArgsOntoSendTxArgs` ([https://github.com/status-im/status-go/pull/1070/files/312a47167309f5ea6d6582b0b7f51d2f2b7bcb08#diff-57c0d14258fd3623b3dd8f0b36b65775L209](url)). Since it was removed there is no need for testing the gas overriding. I removed it because now that we have one call (`sendTransaction`) instead of two (`sendTransaction` and then `approve`) we can just have all the gas info set in `transactions.SendTxArgs`.